agent-service/agents/payroll/feature.py

```
@@ -1,18 +1,180 @@
+                    "LLM responded in %.2fs, model=%s",
+                    llm_elapsed,
+                    llm_response.get("model"),
+                )
+
+                # Split LLM output into two sections
+                parts = raw_text.split("\n\nRecommendation:\n", 1)
+                if len(parts) != 2:
+                    raise ValueError(
+                        "LLM output missing expected "
+                        "'\\n\\nRecommendation:\\n' separator"
+                    )
```

Ruff (0.15.2)

[warning] Abstract `raise` to an inner function

(TRY301)